biblatex.cfg

```diff
@@ -134,8 +134,9 @@
 %Adding a comma to the type field of a thesis
 \DeclareFieldFormat[thesis]{type}{{#1}\addcomma}
 
+%JV 2020/06/20 removed this field as this does not match the 2019 style guide
```

Comment:
We'll see the git log for this line, we can remove the name and date (checkout `git blame`)